app/models/mixins/assignment_mixin.rb

```diff
@@ -81,17 +81,25 @@ def remove_all_assigned_tos(cat = nil)
   delegate :namespace, :to => :class
 
   module ClassMethods
+    # get a mapping of alert_sets and the tags they are assigned
+    #
+    # the namespace is removed from the front of the alert_set's tag
+    #   alert_set will have:
```

Comment:
Is this method only used for alert sets?  I thought assignment_mixin was more generic than that.  If it's generic, this comment implies that its limited to alert sets.